src/reporters/DashboardReporter.ts

```diff
@@ -57,7 +57,7 @@ export class DashboardReporter implements IReporter {
       const componentWithScore: ComponentDto = {
         component: cwp.component,
         dxScore: { value: dxScoreForComponent, points: dxScorePoints },
-        securityIssues: [],
+        securityIssues: <SecurityIssueDto[]>securityVulnerabilitiesPractice[0]?.practice.data?.statistics?.securityIssues,
```

Comment:
```suggestion
        securityIssues: <SecurityIssueDto[]>securityVulnerabilitiesPractice?.practice.data?.statistics?.securityIssues,
```

Reply:
Why? Method filter is returning array - type `PracticeWithContextForReporter[]`

Reply:
Because `find` already returns only one